src/main/java/io/debezium/connector/jdbc/HibernateJAXBRecordMapper.java

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.jdbc;
+
+import java.io.FileNotFoundException;
+import java.io.FileOutputStream;
+import java.util.Locale;
+
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Marshaller;
+
+import org.apache.kafka.connect.data.Field;
+import org.apache.kafka.connect.data.Schema;
+import org.hibernate.boot.jaxb.hbm.spi.JaxbHbmBasicAttributeType;
+import org.hibernate.boot.jaxb.hbm.spi.JaxbHbmHibernateMapping;
+import org.hibernate.boot.jaxb.hbm.spi.JaxbHbmIdentifierGeneratorDefinitionType;
+import org.hibernate.boot.jaxb.hbm.spi.JaxbHbmRootEntityType;
+import org.hibernate.boot.jaxb.hbm.spi.JaxbHbmSimpleIdType;
+import org.hibernate.type.BigIntegerType;
+import org.hibernate.type.BooleanType;
+import org.hibernate.type.ByteType;
+import org.hibernate.type.DateType;
+import org.hibernate.type.FloatType;
+import org.hibernate.type.IntegerType;
+import org.hibernate.type.LongType;
+import org.hibernate.type.TextType;
+import org.hibernate.type.TimeType;
+import org.hibernate.type.TimestampType;
+
+import io.debezium.data.Bits;
+import io.debezium.time.Date;
+import io.debezium.time.MicroDuration;
+import io.debezium.time.ZonedTime;
+import io.debezium.time.ZonedTimestamp;
+
+/**
+ *
+ */
+public class HibernateJAXBRecordMapper {
+    private static final String DEBEZIUM_DATATYPE_PREFIX = "io.debezium";
+
+    public static final String RESOURCES_PATH = HibernateJAXBRecordMapper.class.getResource("/").getPath();
+    public static final String HIBERNATE_XML_SUFFIX = "hbm.xml";
+    public static final String DEBEZIUM_ID = "dbz_id";
+
+    private final JdbcSinkConnectorConfig config;
+
+    public HibernateJAXBRecordMapper(JdbcSinkConnectorConfig config) {
+        this.config = config;
+    }
+
+    /** Makes {@link JaxbHbmHibernateMapping} from record {@link Schema}
+     *  with table name and entity name of the topic
+     *
+     * @param topic
+     * @param schema
+     * @return
```

Comment:
Either provide values or omit the tags altogether if there isn't anything meaningful to say about them.